packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -1,11 +1,21 @@
 import * as React from 'react';
-import { getId, styled, classNamesFunction, IStyleFunctionOrObject, css } from 'office-ui-fabric-react/lib/Utilities';
+import * as ReactDOM from 'react-dom';
+import {
+  getId,
+  styled,
+  classNamesFunction,
+  IStyleFunctionOrObject,
```

Comment:
IStyleFunctionOrObject doesnt seem to be used in the file. Could you remove it please? Though it existed earlier :)

Reply:
It is used, for styles in the ISelectedPersonaProps. styled is used for the SelectedPersona on line 219